resources/tests/type_checker_tests/MultipleConstructors.obs

```diff
@@ -0,0 +1,14 @@
+main contract C {
```

Comment:
Add comment indicating what error is expected here (and what this test case is testing).